go.mod

```diff
@@ -19,7 +19,7 @@ require (
 	go.opencensus.io v0.24.0
 	go.uber.org/atomic v1.9.0
 	go.uber.org/automaxprocs v1.5.3
-	go.uber.org/zap v1.25.0
```

Comment:
the zap change shouldn't be necessary

Reply:
Should I change it to the previous one or it is ok?

Reply:
yeah - leave it unchanged

Reply:
ok

Reply:
you'll need to undo this line or run

`./hack/update-deps.sh`